Repository/ContainerRepositoryFactory.php

```diff
@@ -50,8 +50,8 @@ public function getRepository(EntityManagerInterface $entityManager, $entityName
             if ($this->container && $this->container->has($customRepositoryName)) {
                 $repository = $this->container->get($customRepositoryName);
 
-                if (! $repository instanceof EntityRepository) {
-                    throw new \RuntimeException(sprintf('The service "%s" must extend EntityRepository (or a base class, like ServiceEntityRepository).', $repositoryServiceId));
+                if (!$repository instanceof ObjectRepository) {
```

Comment:
Sorry for the nitpick, but the space is required due to Doctrine Coding Standard.

Reply:
I also fixed the test! Thanks.